modules/stitching/include/opencv2/stitching.hpp

```diff
@@ -259,6 +259,24 @@ class CV_EXPORTS_W Stitcher
      */
     CV_WRAP Status estimateTransform(InputArrayOfArrays images, InputArrayOfArrays masks = noArray());
 
+    /** @overload */
```

Comment:
Overloads should go after the main function.

http://pullrequest.opencv.org/buildbot/export/pr/19619/docs/d2/d8d/classcv_1_1Stitcher.html#a1bbb3bb8cc7763a0177a05b21c97f794

> It differs from the above function only in what argument(s) it accepts.

Reply:
Done.